tools/test/h5dump/CMakeVFDTests.cmake

```diff
@@ -35,34 +35,34 @@ set (HDF5_SF2_VFD_H5DUMP_FILES
 foreach (vfdtest ${VFD_LIST})
   if (vfdtest STREQUAL "subfiling")
     foreach (h5_tfile ${HDF5_SF_VFD_H5DUMP_FILES})
-      file(COPY "${PROJECT_SOURCE_DIR}/testfiles/${h5_tfile}" DESTINATION "${PROJECT_BINARY_DIR}/${vfdtest}")
+      file(COPY "${HDF5_TOOLS_TST_DIR}/h5dump/testfiles/${h5_tfile}" DESTINATION "${PROJECT_BINARY_DIR}/${vfdtest}")
```

Comment:
The situation's less clear for the paths here though.

Reply:
We now have a bit of an awkward situation where most of the test files are in the shared location, some are still in tool-specific sub-directories and some are split between the old and new location, as in the case of the subfiling test files.